homeworks/Aleksandr.Turuta_turuta7/homework_7/homework.test.js

```diff
@@ -0,0 +1,19 @@
+const { cycleAndTest } = require('./homework_7');
+
+describe('test fibonacci', () => {
+    test('test fibonacci', () => {
+        expect(cycleAndTest(0)).toEqual([]);
```

Comment:
again, function naming... why don't name your function by actually what this function is doing?

`makeFiboArr()`, `fibolizer()`... but not `cycleAndTest`...

also, I would make a param this function accepts optional by assigning default value^ so you could run your tests not providing any arguments